cmd/adsysd/integration_tests/adsysctl_policy_test.go

```diff
@@ -451,7 +451,7 @@ func TestPolicyUpdate(t *testing.T) {
 			initState: "old-data",
 			krb5ccNamesState: []krb5ccNamesWithState{
 				{
-					src:          "ccache_EXAMPLE.COM",
+					src:          "ccache_OFFLINE",
```

Comment:
waow, nicely spotted! (or thanks for the test to uncover a future bug :p). I was puzzled for a short while, until I opened the corresponding sssd.conf and confirm with your commit comments :)